modules/mysql/variables.tf

```diff
@@ -30,6 +30,12 @@ variable "random_instance_name" {
   default     = false
 }
 
+variable "database_version_replica" {
+  description = "The read replica database version to use. There is no need to set this, the master database version will be taken by default. However to update databases, read replicas must be updated first, setting this to an update version will cause the replica to update, then you may update the master with the var database_version and leave or remove this field after"
```

Comment:
You can reduce the description here. E.g. the fact that you a `default` for this implies that this is optional.

Reply:
Sounds good!